app/controllers/providers/limitations_controller.rb

```diff
@@ -9,7 +9,7 @@ def show
     end
 
     def update
-      if Setting.allow_multiple_proceedings? && @legal_aid_application.used_delegated_functions?
+      if @legal_aid_application.used_delegated_functions?
```

Comment:
Hi, the request test needs updating to ignore the whole MP flag etc.

Reply:
I have added a test to the section where MP flag is set to false. So we test on both MP and Single proc as they generate different pages, again when MP is rolled out we can remove the legacy one and the associated tests.